### ISSUE TITLE
Can't import the ES module in Angular

### ISSUE THREAD

**Issue by ozkoidi** [gentooboontoo/js-quantities#142]
Hello,

I'm trying to import the ES module version of the library in an Angular 15 project and I get the following error:

![image](https://github.com/gentooboontoo/js-quantities/assets/42638799/f824198c-24f9-4e3b-9c39-e0714ccf514a)

I see that there is an export in [package.json](https://github.com/gentooboontoo/js-quantities/blob/master/package.json) but it doesn't seem to work.

Aditional info: I installed the dependency `"js-quantities": "^1.8.0"` and the devDependency `@types/js-quantities": "^1.6.3"`

**Comment by gentooboontoo** [comment]
Hi. Could you try `import Qty from 'js-quantities';`?

**Comment by ozkoidi** [comment]
Yes, that works but it doesn't use the ES module version right? Which is the one I'm trying to use

**Comment by godind** [comment]
Same here!

**Comment by godind** [comment]
To add more valuable context: using `import Qty from 'js-quantities';` works fine but I get the following...obvious I guess!

> Warning: /.../.service.ts depends on 'js-quantities'. CommonJS or AMD dependencies can cause optimization bailouts.
> For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
